x/oracle/keeper/historic_price.go

```diff
@@ -314,6 +314,17 @@ func (k Keeper) DeleteHistoricMedianDeviation(
 	store.Delete(types.KeyMedianDeviation(denom, blockNum))
 }
 
+// ClearHistoricMedians iterates through all medians in the store and deletes
+// them.
+func (k Keeper) ClearHistoricPrices(ctx sdk.Context) {
```

Comment:
we don't need to export this, especially if it's just for testing
```suggestion
func (k Keeper) clearHistoricPrices(ctx sdk.Context) {
```

Reply:
The test is outside the keeper so this would break it. I moved the clear methods into the test and removed them from the keeper instead.

Reply:
that works ok yeah 💯